flask_app/models/task.py

```diff
@@ -0,0 +1,96 @@
+from flask_app.config.mysqlconnection import connectToMySQL
+from flask import flash
+from flask_app.models.user import User
+from flask_app.models.project import Project
+
+class Task:
+    db = "group_project"
+    def __init__(self,data):
+        self.id = data['id']
```

Comment:
Should a task also have a user_id and a task_id as well? 

![image](https://user-images.githubusercontent.com/107015116/215348246-f76d8d7a-b2db-4da9-8408-b609513ed091.png)

Reply:
thats something I wasn't sure of how we wanted. It almost makes sense to me to take out the user_id in tasks just leave the project_id with the task "id" so that only projects is the middle between tasks and users. I would have to go in and just change the user_id stuff to project_id but I dont see why that wouldn't work.

Reply:
IMO we should just keep both because it technically is a many-to-many (even though at the moment we aren't displaying all of a user's tasks, but we could do something where a user could eventually all their to-dos ordered by project date!). A user can have many tasks and a project can have many tasks. I think having project_id will be most important because we will want to associate tasks with projects when we serve up a single project page.

The attached image of ERD is the way I think we should go. @Teknicl @cmarshall97  thoughts?

Reply:
When we met I questioned this too but couldn't see how it would complicate our code.  We would simply not denote or call it and leave for future use.  However creating it when we need it is just as easy.  I'm in favor of leave it for now, what about you @cmarshall97.

Reply:
As long as we have project_id, I'm fine with it (a project can have many tasks). I imagine we would have to remove the connection to users in the ERD?

Reply:
Yeah thats what I think. I can then update everything in the task from user to project which will base all the tasks off of the project id and the the project will be based off of the user only

Reply:
Sounds good! I'm trying to build form for creating task so that way tomorrow we have something to visually see as we discuss functionality. 

When you feel comfortable, I may ask for permission to play around in back to just so I can trouble shoot quickly tomorrow as I'm trying to make things work on the front-end.